.github/CONTRIBUTING.md

```diff
@@ -1,5 +1,4 @@
 # Contributing
 
-## Request for changes/ Pull Requests
-
-You first need to create a fork of the [meteors](https://github.com/xai4space/meteors) repository to commit your changes to it. When you have made your changes, you can open a pull request on the [meteors repository](https://github.com/xai4space/meteors/pulls).
+Bonjorno My friend! Thank you for considering contributing to Meteors!
```

Comment:
I would use Hi instead of Bonjourno (especially because it is spellee Buongiorno xd)

Reply:
WHYYYYYYY, but ok